app/views/events/_pinned_transactions.html.erb

```diff
@@ -12,7 +12,7 @@
       <% end %>
 
       <%= link_to organizer_signed_in? ? pin.hcb_code : "#", data: @show_popovers ? { turbo_frame: "_top", behavior: "modal_trigger", modal: "transaction_details_#{pin.__id__}" } : {}, class: "pinned__transaction relative card shadow-none border flex flex-col black" do %>
-        <%= form_with url: pin_hcb_code_path(pin.hcb_code, event: @event), model: @hcb_code, data: { turbo_method: :post, turbo_confirm: "This will unpin the transaction for all team members. Continue?" } do |form| %>
+        <%= form_with url: pin_hcb_code_path(pin.hcb_code, event: @event), model: false, data: { turbo_method: :post, turbo_confirm: "This will unpin the transaction for all team members. Continue?" } do |form| %>
```

Comment:
why wouldn't we use `pin.hcb_code`?

Reply:
Given that we're passing in `url` I'm not sure what difference it would make but I can test it out

Reply:
Updated to use a `form_for`